lookout/style/format/analyzer.py

```diff
@@ -125,13 +129,17 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
         :return: List of comments.
         """
         comments = []
-        for fix_data in self.generate_fixes(data_service, list(data["changes"])):
-            text = fix_data.error or self.render_comment_text(fix_data)
-            comments.append(generate_comment(
-                text=text,
-                filename=fix_data.head_file.path,
-                line=fix_data.line_number,
-                confidence=fix_data.confidence))
+        for file_fix in self.generate_file_fixes(data_service, list(data["changes"])):
+            filename = file_fix.head_file.path
+            if file_fix.error:
+                comments.append(generate_comment(text=file_fix.error, line=0, confidence=100,
+                                                 filename=filename))
+
+            else:
+                comments.extend(generate_comment(
+                    text=self.render_comment_text(line_fix, file_fix), filename=filename,
```

Comment:
Can you please replace line_fix with an index inside file_fix? Otherwise, it is possible to pass a line fix which is not contained in the file fix.